addons/map_gestures/stringtable.xml

```diff
@@ -349,6 +360,7 @@
         </Key>
         <Key ID="STR_ACE_Map_Gestures_briefingMode_Proximity">
             <English>Proximity</English>
+            <German>Umgebung</German>
```

Comment:
"Proximity" ist "Nähe".

Reply:
Nähe würde in dem Kontext meiner Meinung nach nicht viel Sinn ergeben
![image](https://user-images.githubusercontent.com/33136944/102808004-f4cfb680-43bf-11eb-888a-6d00f85d198b.png)